src/test/java/duke/task/DeadlineTest.java

```diff
@@ -0,0 +1,14 @@
+package duke.task;
+
+import java.time.LocalDate;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlineTest {
+    @Test
+    public void testDeadline() {
```

Comment:
maybe adding the input type to the function name will be more adhering to the module code standards?